### DISCUSSION TITLE
Title for a widget

### DISCUSSION THREAD

**Discussion by FanaticExplorer** [TomSchimansky/CustomTkinter#1727] [Ideas]
I am creating my first window program using CustomTkinter. Right now, I need to tile every widget with his own label, what is not so comfortable. I think about making a widget which will take another widget as an argument and become a title to it. Also, it could take arguments, which will define, will the widget be outlined, where should title be, and so on.

**Comment by abhaymakes** [comment]
Are you looking for something like [this](https://www.google.com/amp/s/www.geeksforgeeks.org/python-tkinter-create-labelframe-and-add-widgets-to-it/amp/)

**Reply by FanaticExplorer** [reply]
Yes!

**Reply by ghost** [reply]
Accept the answer if it did help you :)

**Reply by abhaymakes** [reply]
@dishb damn, they posted the question in ideas not in Questions, well atleast it helped xD
